pkg/repositories/gormimpl/task_repo.go

```diff
@@ -42,9 +42,6 @@ func (r *TaskRepo) Get(ctx context.Context, input interfaces.Identifier) (models
 		},
 	}).Take(&task)
 	timer.Stop()
-	if tx.Error != nil {
-		return models.Task{}, r.errorTransformer.ToFlyteAdminError(tx.Error)
-	}
```

Comment:
Was gorm error overriding the recordNot found and was that the reason you moved it to L53?

Reply:
yes, exactly.